packages/client/src/schedule-client.ts

```diff
@@ -377,24 +374,22 @@ export class ScheduleClient extends BaseClient {
       );
 
       for (const raw of response.schedules ?? []) {
-        if (!raw.info?.spec) continue;
-
         yield <ScheduleSummary>{
           scheduleId: raw.scheduleId,
 
-          spec: decodeScheduleSpec(raw.info.spec),
-          action: {
+          spec: decodeScheduleSpec(raw.info?.spec ?? {}),
```

Comment:
Probably better to return undefined instead of an empty spec.

Reply:
Doing that means that using and updating a schedule become significantly more complex. For example

```
      await handle.update((x: ScheduleUpdateOptions) => {
        x.spec.intervals = [{ every: '3h' }];
        return x;
      });
```

would now have to be written as

```
      await handle.update((x: ScheduleUpdateOptions) => {
        x.spec = x.spec ?? {};
        x.spec.intervals = [{ every: '3h' }];
        return x;
      });
```

Reply:
Making `spec` optional was actually my first attempt. I really didn't like the consequences of that.

Reply:
Also, the server might actually return a schedule with a spec that has no calandar and no intervals.